### PR TITLE
[Android] Use the locale from system by default

### PR DESCRIPTION
**Pull request by fujunwei**
The browser can use the locale via system by default and the --lang command line flag.

BUG=XWALK-2132

### CODE REVIEW

**Comment by halton** [comment]
Does this change also affect  Android? https://lists.crosswalk-project.org/pipermail/crosswalk-help/2015-March/000913.html is asking same ability.

**Comment by fujunwei** [comment]
Maybe. I will reply it after the PR is merged

**Comment by fujunwei** [comment]
@halton @lincsoon @axinging  PTAL. Thanks

**Comment by halton** [comment]
LGTM, but you need work together with @rakuco to fix the trybot failures.

**Comment by axinging** [comment]
LGTM

File: runtime/browser/xwalk_content_browser_client.cc [hunk 1]
```diff
@@ -429,4 +430,8 @@ content::DevToolsManagerDelegate*
   return new XWalkDevToolsDelegate(browser_context_);
 }
 
+std::string XWalkContentBrowserClient::GetApplicationLocale() {
+  return base::android::GetDefaultLocale();
```
Comment (rakuco) [inline runtime/browser/xwalk_content_browser_client.cc]: Isn't this Java-specific?

**Comment by halton** [comment]
@fujunwei why delete your branch, thus this PR is closed.

**Comment by fujunwei** [comment]
I confuse why it can't compiled successfully.

**Comment by fujunwei** [comment]
i want to send a new PR instead of this.

**Pull request closed without merging**